test/suites/common-tanssi/services-payment/test_services_payment_block_credit_buying_free_combined.ts

```diff
@@ -11,7 +11,7 @@ describeSuite({
     testCases: ({ it, context }) => {
         let polkadotJs: ApiPromise;
         let alice: KeyringPair;
-        const blocksPerSession = 10n;
+        const blocksPerSession = 5n;
```

Comment:
We have `blocksPerSession = 10` because of the 6s block time, right? If this test fails we should divide by 2 where needed, but not here.